js/removeBook.js

```diff
@@ -0,0 +1,14 @@
+export default function removeBook(e, list) {
+  // const newCollectionOfBooks = [];
+
+  if (e.target.matches('.remove')) {
+    const author = e.srcElement.previousElementSibling.textContent;
+
+    const newCollectionOfBooks = list.filter((book) => book.author !== author);
```

Comment:
- [ ] Great job for fixing some requirements and it is working if I delete one book but if the author is the same it deleting all the books but that's not good for user experience let's assume that the same author have different books but the user want to delete one book, should it be deleted all the books that have the same author name??? so please let's chnage it to filter by ID instead of author it is better 🤩💯✅🔥✨ or if you want to filter by author so you can make it unique so the user can't enter the same name you can follow the solution that fits with you! ✨🔥✅💯🤩

Reply:
Done!
I used filter to remove the book from the array